server/views/pages/overview/partials/overviewTab/overviewTabPostInduction.njk

```diff
@@ -3,103 +3,8 @@
 <section data-qa="post-induction-overview">
   {% if not actionPlan.problemRetrievingData %}
 
-    <div class="moj-page-header-actions">
```

Comment:
I've pulled out the action button header and goals list into includes (above) because they will also now be used in the pre-induction overview page (below)